db/migrate/20161103101916_create_applications.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :applications do |t|
       t.string :motivation
       t.references :user, index: true, foreign_key: true
-      t.references :workshop, index: true, foreign_key: true
+      t.references :event, index: true, foreign_key: true
```

Comment:
Please PLEASE read [this](http://edgeguides.rubyonrails.org/active_record_migrations.html#changing-existing-migrations), especially chapters 4, 5 & 6! @Lando-L